### PR TITLE
feat: add unstable_overrides to next

### PR DESCRIPTION
**Pull request by juliusmarminge**
let these options be defined in `createTRPCNext` too

ref: https://discord.com/channels/867764511159091230/1032031459852156938

### CODE REVIEW

File: packages/next/src/withTRPC.tsx [hunk 1]
```diff
@@ -90,7 +92,9 @@ export function withTRPC<
     ssrContext: TSSRContext;
   };
   return (AppOrPage: NextComponentType<any, any, any>): NextComponentType => {
-    const trpc = createReactQueryHooks<TRouter, TSSRContext>();
+    const trpc = createReactQueryHooks<TRouter, TSSRContext>({
+      unstable_overrides: opts.config({}).unstable_overrides,
```
Comment (juliusmarminge) [inline packages/next/src/withTRPC.tsx:96]: this is a bit hacky, not sure what the best approach is to pass the options down?

File: packages/next/src/createTRPCNext.tsx [hunk 2]
```diff
@@ -15,7 +15,10 @@ export function createTRPCNext<
   TRouter extends AnyRouter,
   TSSRContext extends NextPageContext = NextPageContext,
 >(opts: WithTRPCNoSSROptions<TRouter> | WithTRPCSSROptions<TRouter>) {
-  const hooks = createHooksInternal<TRouter, TSSRContext>();
+  const unstable_overrides = opts.config({}).unstable_overrides;
+  const hooks = createHooksInternal<TRouter, TSSRContext>({
+    unstable_overrides,
+  });
```
Comment (juliusmarminge) [inline packages/next/src/createTRPCNext.tsx:21]: bit hacky. maybe these options should be outside the config-callback?

File: packages/next/src/withTRPC.tsx (hunk 1 above)
Comment (juliusmarminge) [reply packages/next/src/withTRPC.tsx:96]: maybe they aren't even needed here? is the hooks created in `createTRPCNext`(see above) the important ones?

Comment (KATT) [reply packages/next/src/withTRPC.tsx:96]: This is fine 🤷

**Review by KATT — APPROVED** [review]

**Pull request merged**